ndk-glue/src/lib.rs

```diff
@@ -16,6 +16,18 @@ use std::thread;
 
 pub use ndk_macro::main;
 
+/// `ndk-glue` macros register the reading end of an event pipe with the
+/// main [`ThreadLooper`] under this ident `ident`.
```

Comment:
minor: should this `ident` be here twice?

Reply:
Whoops, that's sloppy of me. Thanks for noticing!